src/etc/inc/certs.inc

```diff
@@ -312,7 +314,7 @@ function cert_import(& $cert, $crt_str, $key_str) {
 	return true;
 }
 
-function cert_create(& $cert, $caref, $keylen, $lifetime, $dn, $type = "user", $digest_alg = "sha256") {
+function cert_create(& $cert, $caref, $keylen, $lifetime, $dn, $type = "user", $digest_alg = "sha256", $eckey = "brainpoolP160r1", $keytype = "RSA") {
```

Comment:
I'd prefer to see `$keytype` first, and then `$ecname` (not `$eckey`, see above comment) after, but that's not a hard requirement.

Reply:
fixed